BAPSServerAssembly/BAPSController.h

```diff
@@ -207,6 +207,13 @@ namespace BAPSServerAssembly
 				{
 					System::Byte data = serialPort->ReadByte();
 					handled=false;
+					if (data == 255)
```

Comment:
0xff?

Reply:
Hmm, I left it as 255 because all the rest of the button codes are in the UI as ints.